packages/table/src/interactions/draggable.tsx

```diff
@@ -10,6 +10,7 @@ import * as PureRender from "pure-render-decorator";
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
+import { Utils } from "../common/utils";
```

Comment:
That's fair, I guess I've always structured utils as an object. But I see your point. For another PR, though